todo.py

```diff
@@ -1,19 +1,42 @@
 
-def print_todos(todos):
-    if len(todos) == 0:
+todo_list = {
+    '0': {
+    'title': 'do the thing',
+    'completed': False
+    }
+}
+
+def print_todos():
+    complete_count = 0
+    for index in todo_list:
+            if todo_list[index]['completed'] != True:
```

Comment:
Totally valid and correct to compare to `True`! 
But maybe consider doing this instead:
```
if not todo_list[index]['completed']:
```